ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentRelations/DocumentSiblings.js

```diff
@@ -0,0 +1,58 @@
+import { Error, Loader } from '@components';
+import { RelationMultipart, RelationSerial } from '.';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container, Label, Tab } from 'semantic-ui-react';
+
+export default class DocumentSeries extends Component {
+  render() {
+    const { isLoading, error } = this.props;
+
+    const languages = this.props.relations['language'] || [];
+
+    const panes = [
+      {
+        menuItem: {
+          key: 'languages',
+          content: (
+            <>
+              Languages <Label>{languages.length}</Label>{' '}
+            </>
+          ),
+        },
+        render: () => <RelationMultipart />,
+      },
+      // {
```

Comment:
Is this used?

Reply:
no, but I want to make this change in steps, don't worry I will clean everything in the last PR